met-api/src/met_api/utils/enums.py

```diff
@@ -21,6 +21,7 @@ class ContentType(Enum):
     JSON = 'application/json'
     FORM_URL_ENCODED = 'application/x-www-form-urlencoded'
     PDF = 'application/pdf'
+    FORM = 'application/x-www-form-urlencoded'
```

Comment:
Why are we creating `FORM` with the same value as `FORM_URL_ENCODED`? Do we plan to change this later or can we reuse `FORM_URL_ENCODED`

Reply:
Thanks for catching this. I have updated the code to use FORM_URL_ENCODED.